docs/en/observability/synthetics-reference/lightweight-config/http.asciidoc

```diff
@@ -0,0 +1,305 @@
+:hardbreaks-option:
+
+[%header]
+|===
+| Option (type) | Description
+
+// hosts
+| [[monitor-http-hosts]] *`hosts`*
+(list of <<synthetics-lightweight-data-string,string>>s)
+a| *Required*. A list of URLs to ping.
+
+////////////////////////
+max_redirects
+////////////////////////
+| [[monitor-http-max_redirects]] *`max_redirects`*
+(<<synthetics-lightweight-data-numbers,number>>)
+a| The total number of redirections Heartbeat will follow. By default, Heartbeat will not follow redirects, but will report the status of the redirect. If set to a number greater than `0`, Heartbeat will follow that number of redirects.
```

Comment:
Preview: [`max_redirects`](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#monitor-http-max_redirects)

Should the mention of "Heartbeat" here be removed? If yes, what's the right alternative?

```suggestion
a| The total number of redirections Synthetics will follow. By default, Synthetics will not follow redirects, but will report the status of the redirect. If set to a number greater than `0`, Synthetics will follow that number of redirects.
```